caiman/base/movies.py

```diff
@@ -671,7 +671,7 @@ def NonnegativeMatrixFactorization(self,
         T, h, w = self.shape
         Y = np.reshape(self, (T, h * w))
         Y = Y - np.percentile(Y, 1)
-        Y = np.clip(Y, 0, np.Inf)
+        Y = np.clip(Y, 0, np.inf)
```

Comment:
@pgunn I'm almost certain that the method this is in is actually no longer used for anyting in caiman. Lots of methods on `movie` from back when they were trying to implement basically the entirety of caiman in the movie class

Reply:
There are no calls to it in the repo; you're right.

After I merge this I'll go through the movie class and look for more things we can zap.